ledger/src/blockstore_processor.rs

```diff
@@ -3560,38 +3560,10 @@ pub mod tests {
             assert!(result.is_ok()); // No failing transaction error - only instruction errors
             bank.freeze();
 
-<<<<<<< HEAD
             assert_eq!(blockhash_ok, bank.last_blockhash());
-            assert!(bankhash_ok != bank.hash());
+            assert_eq!(bankhash_ok == bank.hash(), fee_payer_in_rent_partition);
```

Comment:
So the merge conflict needed to bring this check from c5473e4b3, but keep the rest from `v2.1` head